Makefile

```diff
@@ -182,15 +182,19 @@ venv_osx:
 clean_venv:
 	rm -rf venv
 
+pip_dev_deps:
+	# Install the depencies for analyze, web and the tools.
+	make -C $(CC_ANALYZER) pip_dev_deps && \
```

Comment:
We are using this part of code multiple times. See `venv_dev` target.

I think it would be better to create a variable to it and us it:
```makefile
PIP_DEV_DEPS_CMD = make -C $(CC_ANALYZER) pip_dev_deps && \
  make -C $(CC_WEB) pip_dev_deps && \
  make -C $(CC_TOOLS)/plist_to_html pip_dev_deps
```